### PR TITLE
✨Add RnetPlus ad exchange support to amp-ad

### PR DESCRIPTION
**Pull request by vkrapivin-sc**
Add RnetPlus ad exchange support to amp-ad

This is yet another Ad Network Exchange from Rambler&Co (also known as RNet+ or RNetPlus); technically it is Russian news exchange service across Russian big sites.

### CODE REVIEW

**Comment by vkrapivin-sc** [comment]
I signed it!

**Comment by mrjoro** [comment]
Sorry for the delay in reviewing this @vkrapivin-sc!  @lannka can you take a look?

/cc @ampproject/wg-ads

File: ads/_config.js [hunk 1]
```diff
@@ -838,6 +838,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'rnetplus': {},
```
Comment (calebcordry) [inline ads/_config.js]: FYI, preload/prefetch are popular options here if you know your creative will be requesting a certain file

File: ads/rnetplus.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/rnetplus.js]: nit: 2019

File: ads/rnetplus.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/rnetplus.md]: nit: 2019

File: examples/ads.amp.html [hunk 4]
```diff
@@ -1580,6 +1581,13 @@ <h2>Rambler&Co</h2>
       json='{"params":{"p1":"bvpkq","p2":"y","pct":"a"}}'>
   </amp-ad>
 
+  <h2>Rambler&Co RNetPlus</h2>
+  <amp-ad width="300" height="400"
+      type="rnetplus"
+      layout="responsive"
+      src="https://api.rnet.plus/Scripts/rnet_amp_embed.js?blockId=660">
```
Comment (calebcordry) [inline examples/ads.amp.html]: It is up to you but you may want to hardcode this url, and have the block-id be a separate attribute? May be less likely for your users to break.

Comment (vkrapivin-sc) [reply examples/ads.amp.html]: Actually, it is little bit `corporate specific`, I am sorry. 

In few words, without RNetPlus team, you have no chances to place certain `block` to your pages with desired content; the empty blocks displays nothing. So, from the other side of this tube, somebody need to collect couple of teasers to display and assign to the particular block. 

As result, RNetPlus Team is always involved into embedding the teasers and it provides the code snippets to place teasers for AMP case and for casual web cases. 

It means the exact form is not important.

But if the form will be important for somebody I able to redesign this piece, thank you.

File: ads/rnetplus.js (hunk 2 above)
Comment (vkrapivin-sc) [reply ads/rnetplus.js]: Fixed

File: ads/rnetplus.md (hunk 3 above)
Comment (vkrapivin-sc) [reply ads/rnetplus.md]: Fixed

File: ads/_config.js (hunk 1 above)
Comment (vkrapivin-sc) [reply ads/_config.js]: Thank you, but from the technical manner we have JSONP request and it cannot be precached. The images and other stuff are relied to server response and we cannot prefetch it too... Anyway I'll review code to think if we still have something to declare, thank you.

Comment (calebcordry) [reply ads/_config.js]: Sounds good.

File: examples/ads.amp.html (hunk 4 above)
Comment (calebcordry) [reply examples/ads.amp.html]: SGTM.

**Comment by calebcordry** [comment]
@vkrapivin-sc looks like you have some formatting errors. Can you run this command `gulp lint --local_changes --fix` and see if the output is correct and commit? 

You can then run `gulp lint --local_changes` and make sure it passes. Let me know if you need any help. Thanks

**Comment by vkrapivin-sc** [comment]
> Can you run this command `gulp lint --local_changes --fix` and see if the output is correct and commit?

Hmm, I've checked this some time; but I see no any errors / warnings for ads/ folder. Seems I need some hints.

**Comment by calebcordry** [comment]
@vkrapivin-sc do you have any output when you run `gulp lint`? Here is the output I am seeing on travis : 
Screen Shot 2019-07-17 at 9 39 47 AM

Looks like your editor is inserting carriage returns, and prettier wants you to remove them.

**Comment by vkrapivin-sc** [comment]
> Looks like your editor is inserting carriage returns, and prettier wants you to remove them.

Got it, thank you. It seems dos2unix conversion helped. Sorry for this mistake.

**Comment by calebcordry** [comment]
No problem, thanks for contributing!

**Pull request merged**